#
# request 
#
bool command_request
---
#
# response: 1--> quiz; 2--> system; 3--> stop
#
int8 command_response
int8 command_quiz
